kpops/components/base_components/kafka_connector.py

```diff
@@ -94,6 +94,18 @@ class KafkaConnector(PipelineComponent, ABC):
     class Config(CamelCaseConfig):
         pass
 
+    @validator("app", pre=True)
+    def connector_config_should_have_component_name(
+        cls, v: KafkaConnectorConfig | dict[str, str], values: dict[str, Any]
```

Comment:
```suggestion
        cls, kafka_connect_config: KafkaConnectorConfig | dict[str, str], values: dict[str, Any]
```

Reply:
wouldn't it be better to rename it to `app` since that's the name of the attribute?